arangodb-net-standard/DocumentApi/DocumentApiClient.cs

```diff
@@ -286,5 +287,43 @@ public async Task<DeleteDocumentsResponse<T>> DeleteDocumentsAsync<T>(string col
             }
         }
 
+
+
+        /// <summary>
+        /// Get headers based on collection and _key with option headers.
+        /// 200: is returned if the document was found. 
```

Comment:
Maybe put that in the `remarks`? It seems a bit too much to include in the summary, imo.